glue_job.tf

```diff
@@ -26,45 +20,60 @@ resource "aws_glue_crawler" "signatures_crawler" {
 
 resource "aws_s3_bucket" "glue_resources" {
   bucket = var.glue_scripts_bucket_name
+}
 
-  acl = "private"
-  server_side_encryption_configuration {
-    rule {
-      apply_server_side_encryption_by_default {
-        sse_algorithm = "AES256"
-      }
-    }
-  }
+resource "aws_s3_bucket_acl" "glue_resources" {
+  bucket = aws_s3_bucket.glue_resources.id
+  acl    = "private"
+}
 
-  lifecycle_rule {
```

Comment:
Seems like we would want to keep this rule for deleting temp files?

Reply:
Similarly we added this manually added lifecycle rule to the config: https://github.com/MoveOnOrg/terraform-aws-controlshift-redshift-sync/commit/19be449c3420f05b3f590e2baa8611d59472ae67 and https://github.com/MoveOnOrg/terraform-aws-controlshift-redshift-sync/commit/7fa1de1bf76bd0739438572e8cfcea5c1b6b4113